native_client/javascript/abi_crosswalk_priv.json

```diff
@@ -0,0 +1,2206 @@
+    "node_abi": 67,
+    "v8": "7.0"
+  },
+  "11.14.0": {
+    "node_abi": 67,
+    "v8": "7.0"
+  },
+  "11.15.0": {
+    "node_abi": 67,
+    "v8": "7.0"
+  },
+  "12.0.0": {
+    "node_abi": 72,
+    "v8": "7.4"
+  },
+  "12.1.0": {
+    "node_abi": 72,
+    "v8": "7.4"
+  },
+  "12.2.0": {
+    "node_abi": 72,
+    "v8": "7.4"
+  },
+  "12.3.0": {
+    "node_abi": 72,
+    "v8": "7.4"
+  },
+  "12.3.1": {
+    "node_abi": 72,
+    "v8": "7.4"
+  },
+  "12.4.0": {
+    "node_abi": 72,
+    "v8": "7.4"
+  },
+  "12.5.0": {
+    "node_abi": 72,
+    "v8": "7.5"
+  },
+  "12.6.0": {
+    "node_abi": 72,
+    "v8": "7.5"
+  },
+  "12.7.0": {
+    "node_abi": 72,
+    "v8": "7.5"
+  },
+  "12.8.0": {
+    "node_abi": 72,
+    "v8": "7.5"
+  },
+  "12.8.1": {
+    "node_abi": 72,
+    "v8": "7.5"
+  },
+  "12.9.0": {
+    "node_abi": 72,
+    "v8": "7.6"
+  },
+  "12.9.1": {
+    "node_abi": 72,
+    "v8": "7.6"
+  },
+  "12.10.0": {
+    "node_abi": 72,
+    "v8": "7.6"
+  },
+  "12.11.0": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.11.1": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.12.0": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.13.0": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.13.1": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.14.0": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.14.1": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.15.0": {
+    "node_abi": 72,
+    "v8": "7.7"
+  },
+  "12.16.0": {
+    "node_abi": 72,
+    "v8": "7.8"
+  },
+  "12.16.1": {
+    "node_abi": 72,
+    "v8": "7.8"
+  },
+  "12.16.2": {
+    "node_abi": 72,
+    "v8": "7.8"
+  },
+  "12.16.3": {
+    "node_abi": 72,
+    "v8": "7.8"
+  },
+  "13.0.0": {
+    "node_abi": 79,
+    "v8": "7.8"
+  },
+  "13.0.1": {
+    "node_abi": 79,
+    "v8": "7.8"
+  },
+  "13.1.0": {
+    "node_abi": 79,
+    "v8": "7.8"
+  },
+  "13.2.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.3.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.4.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.5.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.6.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.7.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.8.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.9.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.10.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.10.1": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.11.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.12.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.13.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "13.14.0": {
+    "node_abi": 79,
+    "v8": "7.9"
+  },
+  "14.0.0": {
+    "node_abi": 83,
+    "v8": "8.1"
+  },
+  "14.1.0": {
+    "node_abi": 83,
+    "v8": "8.1"
+  },
+  "14.2.0": {
+    "node_abi": 83,
+    "v8": "8.1"
+  },
+  "14.3.0": {
+    "node_abi": 83,
+    "v8": "8.1"
+  },
+  "15.0.0": {
+    "node_abi": 84,
```

Comment:
Are you sure this should not be `88` ?

Reply:
CI failing with `Error: Cannot find module '/home/build-user/ds-test/node_modules/deepspeech/lib/binding/v0.9.0-alpha.10/linux-x64/node-v88/deepspeech.node'` and `NODE_MODULE_VERSION` for v15 is `88` on https://nodejs.org/en/download/releases/

Reply:
fixed.